docs/gettingstarted.md

```diff
@@ -271,6 +271,41 @@ $ ./torq.sh start rdb1 -extras -T 60 -w 4000
 $ ./torq.sh start sort1 -extras -s -3
 ```
 
+<a name="profiler">
+
+Using the Code Profiler with torq.sh
+------------------------------------
+
+KDB 4.0 includes an experimental built-in call-stack snapshot primitive that allows
+building a sampling profiler.  The profiler uses the new function [`.Q.prf0`](https://code.kx.com/q/ref/dotq/#qprf0-code-profiler). 
+As of now, this function has the following requirements:
+
+- Must be running KDB+ 4.0 or higher
+- Currently implemented for x86_64 Linux only (kdb+ `l64`).
+- By default, processes can only profile their direct children (*i.e.* started
+with `system"q ..."`) as it is currently linux only
+- The process to be profiled must be started from the same binary as the one 
+running `.Q.prf0` otherwise it will throw a `binary mismatch` error.
+
+Assuming a process is running, you can run the code below as an example in the command line.
+Note that this `top` function currently only allows a single process as an argument and
+multiple processes is not currently supported.
+
+```
+$ ./torq.sh top rdb1
+```
+
+This uses the `top.q` script given by kx (description found [here](https://code.kx.com/q/kb/profiler/))
```

Comment:
Can you make sure there is a line of reference as to where top.q came from, at the top of top.q please.